include/soter/soter_asym_cipher.h

```diff
@@ -61,16 +61,16 @@ soter_asym_cipher_t* soter_asym_cipher_create(const void* key,
 
 /**
  * @brief encrypt data
- * @param [in] asym_cipher_ctx pointer to asymmetric encription/decription context previously
+ * @param [in] asym_cipher_ctx pointer to asymmetric encryption/decryption context previously
  * created by soter_asym_cipher_create
  * @param [in] plain_data data to encrypt
  * @param [in] plain_data_length length of plain_data
  * @param [out] cipher_data buffer for cipher data store. May be set to NULL for cipher data length
  * determination
  * @param [in, out] cipher_data_length length of cipher_data
- * @return result of operation, @ref SOTER_SUCESS on success or @ref SOTER_FAIL on failure
- * @note If cipher_data==NULL or cipher_data_length less then need to store cipher data, @ref
- * SOTER_BUFFER_TOO_SMALL will return and cipher_data_length will contain length of buffer thet need
+ * @return result of operation, @ref SOTER_SUCCESS on success or @ref SOTER_FAIL on failure
+ * @note If cipher_data==NULL or cipher_data_length less then needed to store cipher data, @ref
```

Comment:
If we go around fixing typos...
```suggestion
 * @note If cipher_data==NULL or cipher_data_length less than needed to store cipher data, @ref
```
(the same “then need” mistake is made in a bunch of other places)